hooks/ui_config_advanced.py

```diff
@@ -313,6 +313,11 @@ def get_item_subtitle(self, index):
 
         return subtitle
 
+    @staticmethod
+    def get_item_background_color():
+        """Returns the brush to use draw the background for this widget"""
+        return QtGui.QApplication.palette().light()
```

Comment:
The default background color should not change from what it currently is, so it does not affect other apps using it. Please revert to returning the value:
```
QtGui.QApplication.palette().midlight()
```

If you want to use a different color, you need to override this hook function.

Reply:
True true, I've updated!